greeter/README.md

```diff
@@ -1,21 +1,510 @@
-# Step 1
+# Write scalable and extensible Node.js applications using LoopBack 4
 
-- Create Message type file
-- Create a greeting controller
-- Create a mock greeter service
-- Bind the service to app
-- Try endpoint to print out hello world
+## What is LoopBack?
 
-# Step 2
+[LoopBack](https://loopback.io/) is a popular open source API and Microservice framework 
+in Node.js. Built on top of Express, it enables developers to quickly create REST and other forms of APIs or Microservices to interact with backend databases, services and other infrastructure using model driven design patterns. The command line tool makes the API creation easy and productive. 
 
-- Mount greeter service component
-- Try endpoint with different language settings
-- Create a fr greeter
-- Register it to the app
-- Try endpoint again
+Rewritten in TypeScript from ground-up, LoopBack 4 core offers an Inversion of Control container with powerful Dependency Injection capabilities as the foundation to build large-scale Node.js projects with great flexibility and extensibility. LoopBack also integrates with cloud native health, metrics, and tracing to facilitate development of Microservices for cloud deployment.
 
-# Step 3
+## About This Workshop
 
-- Add caching observer
-- Add caching interceptor
-- Add caching service
\ No newline at end of file
+As your Node.js application becomes more complex, how do you increase its scalability with proven design patterns and best practices? How do you follow 12-factor principles to make your application flexible, extensible and composable? Let’s run a fun workshop to create a scalable and extensible application! While we could do this from scratch, we'll get there much faster and easier using a framework that does a lot of the hard work for us.
+
+In this workshop, you will be guided to scaffold a greeter app using the LoopBack 4 framework, and create a simple greeting API that prints out a welcome message, then enrich it without modifying its implementation in the next 4 steps:
+
```

Comment:
These instructions are pretty clear. I think we might improve the way we introduce it: 
how is it extensible? how is it scalable?

1. show what we (users) have now: some functionalities(`greeter` in our case). 
2. show our expectation (maybe show the intention with an image): an application that has several APIs greets to users with their names and specified language, and has caching server to reduce the cost.
3. explain that which step shows extensibility or flexibility or something else. Example: 
```
in step 1, it shows that LB4 allows to add APIs easily(scalable?) 
in step 2, we can add extensions on the top of it(extensible?) 
in step 3, we can also add caching server to their LB4 app (scalable or entensible?)